vowelReverse.js

```diff
@@ -0,0 +1,5 @@
+const vowelPattern = /[aiuoaeAIUOAE]/g;
```

Comment:
you can make this simple by only writing five vowels and add case-insensitive to the global flag.

e.g: const vowelPattern = /[aiuoe]/gi;

Reply:
sure thanks 🙂